joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/AstCreator.scala

```diff
@@ -497,8 +497,10 @@ class AstCreator(
       })
       .toList
 
-    val stmtAsts = whenThenAstsList ++ Option(ctx.caseExpression().elseClause()).map(astForElseClause).getOrElse(Seq())
-    val block    = blockNode(ctx.caseExpression())
+    val stmtAsts = whenThenAstsList ++ Option(ctx.caseExpression().elseClause())
+      .map(ctx => astForCompoundStatement(ctx.compoundStatement()))
```

Comment:
just a nit
```suggestion
     .map(astForCompoundStatement(_.compoundStatement()))
```

Reply:
I like nits, but it's not the same expansion: your suggestion would expand to (modulo alpha-conversion..)

```scala
.map(astForCompoundStatement(x => x.compoundStatement()))
```

Or am I missing something?

Reply:
ohh, you're right, just saw it. ok nvm